packages/odyssey-react/.postcssrc.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const { default: transformStyles } = require('@okta/odyssey-transform-styles-postcss-preset');
+
+module.exports = (ctx) => {
+  if (!ctx.transformStyles) { return {}; }
```

Comment:
@jeffbelser-okta I'm not sure we need this, maybe I'm misunderstanding. If `!ctx.transformStyles`, don't we still want to return `plugins`?

Reply:
@shawyu-okta great question - My change here is really just about avoiding the use of our very complicated postcss preset when it isn't needed, like in the context of our existing storybook build. In the very near future I DO want to use the preset as part of the storybook build, but for now css modules are loaded using another system (webpack loader pipeline).

To be crystal clear, postcss-load-config will add missing config options [if they are not defined like plugins](https://github.com/postcss/postcss-load-config/blob/main/src/index.js#L32-L34)

Reply:
Got it, thanks for the explanation @jeffbelser-okta